update/Pos/Update/Constants.hs

```diff
@@ -1,13 +1,99 @@
+{-# LANGUAGE TemplateHaskell #-}
+
 module Pos.Update.Constants
-       ( ourAppName
+       ( UpdateConstants(..)
+       , updateConstants
+
+       , ourAppName
        , lastKnownBlockVersion
        , curSoftwareVersion
+
+       -- * Genesis constants
+       , genesisBlockVersion
+       , genesisSoftwareVersions
+       , genesisBlockVersionData
+       , genesisScriptVersion
+       , genesisSlotDuration
+       , genesisMaxBlockSize
+       , genesisMaxHeaderSize
+       , genesisMaxTxSize
+       , genesisMpcThd
+       , genesisHeavyDelThd
+       , genesisUpdateVoteThd
+       , genesisMaxUpdateProposalSize
+       , genesisUpdateProposalThd
+       , genesisUpdateImplicit
+       , genesisUpdateSoftforkThd
        ) where
 
 import           Universum
 
-import           Pos.Core  (ApplicationName, BlockVersion (..), SoftwareVersion (..),
-                            mkApplicationName)
+import           Data.Aeson                 (FromJSON (..), defaultOptions,
```

Comment:
Why `defaultOptions` from aeson? [Here](https://github.com/input-output-hk/cardano-sl/blob/559dc01a946ca2f881e323c285d48e8399195953/src/Pos/Aeson/CompileConfig.hs#L9) we use default options from `serokell-util`, for example. I guess something may be broken because of this change.